typeDefs.js

```diff
@@ -462,9 +462,13 @@ const typeDefs = gql`
     deleteModel(id: ID!): ModelResult
 
     "Contacts Mutation"
-    addContact(contact: contactInput!): ContactResult
+    addContact(contact: contactInput!, upload: Upload!): ContactResult
     deleteContact(id: ID!): ContactResult
-    updateContact(id: ID!, contact: contactInput!): ContactResult
+    updateContact(
+      id: ID!
+      contact: contactInput!
+      upload: Upload!
```

Comment:
Please, rename te upload to the `mapImages` and create custom input `MapImage` that should look like `{ lang: String!, image: Upload! }`, `mapImages` should be an array of that custom type. Also, file uploads should be optional because then tests will fail, we need to think about how to handle file uploads in the test mode